File: app/src/Entity/PackageEntity.java
package Entity;

import javax.persistence.*;

/**
 * Created by arthurn on 24.06.17.
 */
@Entity
@Table(name = "package", schema = "public", catalog = "dav5bc69lqbe93")
public class PackageEntity {
    private int id;
    private String trackNumber;
    private boolean status;

    @Id
    @Column(name = "id", nullable = false)
    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    @Basic
    @Column(name = "track_number", nullable = false, length = -1)
    public String getTrackNumber() {
        return trackNumber;
    }

    public void setTrackNumber(String trackNumber) {
        this.trackNumber = trackNumber;
    }

    @Basic
    @Column(name = "status", nullable = false)
    public boolean isStatus() {
        return status;
    }

    public void setStatus(boolean status) {
        this.status = status;
    }

    @Override
    public boolean equals(Object o) {
        if (this == o) return true;
        if (o == null || getClass() != o.getClass()) return false;

        PackageEntity that = (PackageEntity) o;

        if (id != that.id) return false;
        if (status != that.status) return false;
        if (trackNumber != null ? !trackNumber.equals(that.trackNumber) : that.trackNumber != null) return false;

        return true;
    }

    @Override
    public int hashCode() {
        int result = id;
        result = 31 * result + (trackNumber != null ? trackNumber.hashCode() : 0);
        result = 31 * result + (status ? 1 : 0);
        return result;
    }
}

File: src/main/scala/entities/PackageEntity.scala
package entities

import javax.persistence._

/**
  * Created by arthurn on 24.06.17.
  */
@Entity
@Table(name = "package", schema = "public")
class PackageEntity {
  private var id = 0
  private var trackNumber = ""
  private var status = false
  private var carrier = ""

  @Id
  @Column(name = "id", nullable = false)
  @GeneratedValue(strategy = GenerationType.SEQUENCE)
  def getId: Int = id

  def setId(id: Int): Unit = {
    this.id = id
  }

  @Basic
  @Column(name = "carrier", nullable = false)
  def getCarrier: String = carrier

  def setCarrier(carrier :String): Unit = {
    this.carrier = carrier
  }

  @Basic
  @Column(name = "track_number", nullable = false, length = -1)
  def getTrackNumber: String = trackNumber

  def setTrackNumber(trackNumber: String): Unit = {
    this.trackNumber = trackNumber
  }

  @Basic
  @Column(name = "status", nullable = false)
  def isStatus: Boolean = status

  def setStatus(status: Boolean): Unit = {
    this.status = status
  }

  override def equals(o: Any): Boolean = {
    if (this.equals(o)) return true
    if (o == null || (getClass ne o.getClass)) return false
    val that = o.asInstanceOf[PackageEntity]
    if (id != that.id) return false
    if (status != that.status) return false
    if (if (trackNumber != null) !(trackNumber == that.trackNumber)
    else that.trackNumber != null) return false
    true
  }

  override def hashCode: Int = {
    var result = id
    result = 31 * result + (if (trackNumber != null) trackNumber.hashCode
    else 0)
    result = 31 * result + (if (status) 1
    else 0)
    result
  }
}